apps/cookbook/src/app/showcase/form-field-showcase/form-field-showcase.component.html

```diff
@@ -1,120 +1,78 @@
-<h2>Input Large (default)</h2>
-
+<h2>Input</h2>
+<cookbook-form-field-example-configuration
+  [size]="size"
+  (sizeChanged)="setSize($event)"
```

Comment:
Consider using banana-in-a-box syntax
```suggestion
  [(size)]="size"
```